wcfsetup/install/files/lib/data/user/avatar/DefaultAvatar.class.php

```diff
@@ -56,7 +56,7 @@ public function __construct($username = '') {
 			
 			// the <path> is basically a shorter version of a <rect>
 			$svg = <<<SVG
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="16" height="16"><path fill="#{$backgroundColor}" d="M0 0h16v16H0z"/><text x="8" y="8" fill="#{$textColor}" text-anchor="middle" dy=".3em" font-family="Arial" font-size="7">{$text}</text></svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="128" height="128"><path fill="#{$backgroundColor}" d="M0 0h16v16H0z"/><text x="8" y="8" fill="#{$textColor}" text-anchor="middle" dy=".3em" font-family="Arial" font-size="7">{$text}</text></svg>
```

Comment:
Have you checked this change in IE11? I remember it being very picky on different dimensions for the svg and the `viewBox`?

Reply:
Looking good in Internet Explorer 11

![unknown](https://user-images.githubusercontent.com/4348193/39474853-2e3f2a9e-4d56-11e8-91b4-41e3d02870df.png)